README.md

```diff
@@ -11,10 +11,10 @@ different repository, and only use some of the code and ideas found here.
 make clean
 sudo snap remove charmed-openstack-upgrader --purge
 make build
-sudo snap install ./charmed-openstack-upgrader.snap --devmode
+sudo snap install ./charmed-openstack-upgrader.snap --dangerous
 sudo snap connect charmed-openstack-upgrader:juju-client-observe snapd
-sudo snap connect charmed-openstack-upgrader:dot-local-share-juju snapd
-sudo snap connect charmed-openstack-upgrader:local-cou snapd
+sudo snap connect charmed-openstack-upgrader:dot-local-cou snapd
+sudo snap alias charmed-openstack-upgrader.cou cou
```

Comment:
We should state that this is for local builds. It would be easier when we are ready to add `install from snapstore` section